week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
+            <div class="text">버튼을 눌러 시작하세요!</div>
+        </div>
+        
+        <div class="middle">
+            <div id="human_select"><img id="img" src="./img/paper.png"/></div>
+            <div id="robot_select"><img id="robotimg" src="./img/paper.png"/></div>
+        </div>
+        <div class="bottom">
+            <button onclick="scissor_button()" id="img_toScissor">가위</button>
+            <button onclick="rock_button()" id="img_toRrock">바위</button>
+            <button onclick="paper_button()" id="img_toPaper">보</button>
```

Comment:
이벤트에 관한 내용은 JavaScript와 관련이 있으므로, js 부분에서 한 번에 관리하는 것이 좋습니다.
```suggestion
            <button id="scissor">가위</button>
            <button id="rock">바위</button>
            <button id="paper">보</button>
```